ctapipe/reco/shower_reconstructor.py

```diff
@@ -5,28 +5,34 @@
 from iminuit import Minuit
 
 from ctapipe.containers import Model3DReconstructedGeometryContainer
-from ctapipe.core.traits import Unicode
+from ctapipe.core.traits import FloatTelescopeParameter, Unicode
 from ctapipe.image import (
     GaussianShowermodel,
     ShowermodelPredictor,
     neg_log_likelihood_approx,
 )
-from ctapipe.reco.reco_algorithms import Reconstructor
-
-# taken from ImPACT PR for now
-ped_table = {
-    "LSTCam": 1.4,
-    "NectarCam": 1.3,
-    "FlashCam": 1.3,
-    "CHEC": 0.5,
-    "ASTRICam": 0.5,
-    "dummy": 0.01,
-    "UNKNOWN-960PX": 1.0,
-}
+from ctapipe.reco import Reconstructor
 
 
 class Model3DGeometryReconstructor(Reconstructor):
-    geometry_seed = Unicode(default_value="HillasReconstructor").tag(config=True)
+    geometry_seed = Unicode(
+        default_value="HillasReconstructor",
+        help="GeometryReconstructor that is used as a seed for the likelihood fit",
+    ).tag(config=True)
+
+    pedestal_width = FloatTelescopeParameter(
+        default_value=[
+            ["type", "LST_LST_LSTCam", 1.4],
+            ["type", "MST_MST_FlashCam", 1.3],
+            ["type", "MST_MST_NectarCam", 1.3],
+            ["type", "SST_ASTRI_CHEC", 0.5],
+        ],
+        help="Pedestal charge std",
```

Comment:
call the parameter `pedestal_charge_std` if that is what it is.

Reply:
yes can do

Reply:
I had to remove the default values per type. This fails for a subarray without SSTs for example

Reply:
It's only one value now